external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -167,29 +167,31 @@ class KafkaTestUtils(
    * In this method we rewrite krb5.conf to make kdc and client use the same enctypes
    */
   private def rewriteKrb5Conf(): Unit = {
```

Comment:
@LuciferYang, can you fix this line to `Utils.tryWithResource(Source.fromFile(kdc.getKrb5conf, "UTF-8")) { krb5Conf  =>`? that will reduce the diff  a lot.

Reply:
replace with  `Utils.tryWithResource(Source.fromFile(kdc.getKrb5conf, "UTF-8"))(_.getLines().toList)`, only 1 line changed